src/hooks/Video.js

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+
+import { VideoContext } from '../shared/VideoContext'
+
+const baseClass = 'video'
+
+export const Video = props => {
+  const videoClass =
+    props.size === 'large'
```

Comment:
Maybe this is normal and I'm remembering wrong, but is the first line of the ternary usually on the same line as the variable name?
```js
const videoClass = props.size === 'large'
  ? `${baseClass}__wrapper--large`
  : `${baseClass}__wrapper`
```

Does this project have prettier? Was that just prettier doing its thing?

Reply:
I wish but this is prettier auto-formatting.